ansible/roles/nginx/tasks/main.yml

```diff
@@ -1,18 +1,18 @@
 ---
 
-  - name: Download nginx release
+  - name: Download openresty/nginx bundle
```

Comment:
What happens to the current nginx installation (if any)? Is there a need to remove that first (or perhaps manually)?